### ISSUE TITLE
No rumble in game with evdev

### ISSUE THREAD

**Issue by orbea** [RPCS3/rpcs3#11184]
## Quick summary

When playing Tales of Xillia (BLES01815) using evdev and bluetooth there is no rumble in game. The rumble does work in the rpcs3 input menu.

## Details

When playing Tales of Xillia (BLES01815) there is no rumble in game, but the rumble does work in the rpcs3 input menu when selecting either the large and/or small motors. I am using evdev and bluetooth with a 8BitDo SN30 Pro+ gamepad where rumble does work reliably in other emulators like mupen64plus or other games using wine. I also ensured the rumble is enabled in the in game settings.

Its been a long time since I played this on real hardware, but I am pretty sure its supposed to rumble during the battles? Please correct me if I am doing something wrong. I also have not tested any other games yet.

log:  [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/7577880/RPCS3.log.gz)

#### 5. Please provide your system configuration:
* OS :`Gentoo`
* CPU: `i5-7600K`
* GPU: `Radeon RX 5700 XT`
* Driver version: https://github.com/mesa3d/mesa/commit/14fca01b32958b827dcce63ffa9fd66c039a7cc4
* rpcs3 version: 7dc3a50dd
* libevdev version: `1.11.0`
* xf86-video-evdev version: `2.10.6`

**Comment by RipleyTom** [comment]
Evdev rumble works here. If it works in setting window it will work in-game as the same function is called.
https://gamefaqs.gamespot.com/boards/616012-tales-of-xillia/67277653

Please reopen if you're absolutely sure something is going wrong.

**Comment by orbea** [comment]
Thanks for the link, I failed to find any similar information when I looked.

> When you get hit I believe. I think it also rumbles during a mystic arte if you have multiple people playing to tell who it is doing it. I seem to feel rumble more with multiplayer than single player.

I'm not getting any rumble effects in battles at all, but I'm not far enough in the game for mystic artes yet. I'll have to test more, try other games and possibly spend the time to test it on real HW. I'll update this issue or make a new one when I end up getting around to that.
